### PR TITLE
bug-1554287 - remove zooming effect and add scroll to selected job

### PR DESCRIPTION
**Pull request by loalexandra**
Hi, @sarah-clements and @camd!

Bug is: https://bugzilla.mozilla.org/show_bug.cgi?id=1554287
I've removed zooming effect and added `findJob` [icon](https://fontawesome.com/icons/crosshairs?style=solid) to `ActionBar`

[Here](https://ibb.co/kqgzCRM) is screenshot. 

Let me know If I can do anything else :)

### CODE REVIEW

**Comment by loalexandra** [comment]
I'll fix linter soon :)

**Comment by loalexandra** [comment]
Hi @camd! 

Thank you for review, it's very helpful! 

I returned scaling, but 1.3 still [seems](https://ibb.co/zfj4sGn) too much (overlaps other icons). I've set it to 1.1 (here is [screenshot](https://ibb.co/jLXgt5H)). What do you think about it? 

You're right about `cancel` button. I wide summary panel to 310px. [Here](https://ibb.co/zm1F3Z5) is screenshot of the panel with cancel button. :)

**Comment by sarah-clements** [comment]
@loalexandra Cam won't be back online until Monday to answer your question/approve your pr. While you're waiting, you can finalize your application.

**Comment by camd** [comment]
Thanks for your fixes.  I will review this again on Monday.  I was out on PTO Friday (today).

File: ui/job-view/details/summary/ActionBar.jsx [hunk 1]
```diff
@@ -477,6 +482,21 @@ class ActionBar extends React.PureComponent {
                   </a>
                 </li>
               ))}
+            <li>
+              <Button
+                id="find-job-btn"
+                title="Find this job instance"
```
Comment (camd) [inline ui/job-view/details/summary/ActionBar.jsx]: Preferred wording would be: "Scroll to selection"

File: ui/css/treeherder-job-buttons.css [hunk 2]
```diff
@@ -105,8 +105,13 @@
   border: 2px solid;
   border-radius: 3px;
   font-size: 12px;
-  transform: scale(1.7, 1.7);
   z-index: 1;
+  transition: 0.1s;
+  transform: scale(1.1, 1.1);
```
Comment (camd) [inline ui/css/treeherder-job-buttons.css]: A scale of ``1.1`` is going to be to subtle to pick out from the rest of the job field.  Let's go with ``1.3``.

File: ui/job-view/details/summary/ActionBar.jsx [hunk 3]
```diff
@@ -423,7 +429,6 @@ class ActionBar extends React.PureComponent {
       currentRepo,
     } = this.props;
     const { customJobActionsShowing } = this.state;
-
```
Comment (camd) [inline ui/job-view/details/summary/ActionBar.jsx]: We should leave this blank line.  It's a general convention to have a blank line between ``const``s and functionality.  There are places in the code that haven't been fixed for this yet, but let's stick with it where we can.

File: ui/job-view/details/summary/ActionBar.jsx [hunk 4]
```diff
@@ -477,6 +482,21 @@ class ActionBar extends React.PureComponent {
                   </a>
                 </li>
               ))}
+            <li>
+              <Button
+                id="find-job-btn"
+                title="Find this job instance"
+                className="actionbar-nav-btn btn icon-blue bg-transparent border-0"
+                onClick={() =>
+                  findJobInstance(jobLogUrls[0] && jobLogUrls[0].job_id, true)
```
Comment (camd) [inline ui/job-view/details/summary/ActionBar.jsx]: This is great.  And the new spacing fixes things nicely, I think.  Some day we may want to add a "pulse" of some kind.  Like have it scale up to 2.0, then back to 1.3 when you hit this button.  But don't have to do that now.

File: ui/job-view/details/summary/ActionBar.jsx (hunk 1 above)
Comment (loalexandra) [reply ui/job-view/details/summary/ActionBar.jsx]: Fixed

File: ui/css/treeherder-job-buttons.css (hunk 2 above)
Comment (loalexandra) [reply ui/css/treeherder-job-buttons.css]: Fixed

File: ui/job-view/details/summary/ActionBar.jsx (hunk 3 above)
Comment (loalexandra) [reply ui/job-view/details/summary/ActionBar.jsx]: Fixed

**Comment by loalexandra** [comment]
Hi @camd and @sarah-clements!

Thank you for feedback. I think I fixed it.

**Pull request merged**